Create a shared cover crop council logo displayer #4

ravighevariya10 opened the pull request:
Created a shared CoverCropCouncil logo displayer

ravighevariya10 committed:
Created a shared CoverCropCouncil logo displayer

ravighevariya10 committed:
Added the useEffect() of the favicons and logo images

YazidZidane commented on src/logo-displayer/PSACoverCropCouncil.jsx:
```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useRef } from 'react';
+
+const PSACoverCropCouncil = ({
+  councilShorthandRedux,
```
Can you rename this to `council`?

YazidZidane commented on src/logo-displayer/PSACoverCropCouncil.jsx:
```diff
@@ -0,0 +1,60 @@
+  );
+};
+
+export default PSACoverCropCouncil;
```
Can you also add descriptions to the props using `propTypes`? You can refer to other components for that

YazidZidane commented:
Can you also rename the component to `PSALogoDisplayer` or something like that to make it more meaningful? And also add the component to `src/index.js`

ravighevariya10 committed:
Fixed:
1. Renamed 'councilShorthandRedux' to 'council'
2. Added props description
3. Changed component name to 'PSALogoDisplayer'
4. Added the component in the 'src/index.js' file

YazidZidane commented on src/logo-displayer/PSACoverCropCouncil.jsx:
```diff
@@ -0,0 +1,65 @@
+PSALogoDisplayer.propTypes = {
+  council: PropTypes.string,
+  alt: PropTypes.string,
+  style: PropTypes.object
```
Can you also add descriptions to each props in JSdoc format just like other components?
```
/**
 * add comments here
 */
```

ravighevariya10 replied:
Done @YazidZidane

ravighevariya10 committed:
Added description to the pops

ravighevariya10 committed:
Merge branch 'develop' into covercropcouncil-logo
